docs/credits.md

```diff
@@ -16,21 +16,24 @@ The `Logisim-evolution` project is based on `Logisim` software by:
 The following people and institutions actively contributed to further development of the `Logisim-evolution`:
 
 * David H. Hutchens, [Millersville University](https://www.millersville.edu/), Pennsylvania, USA
-* Kevin Walsh, [College of the Holy Cross](http://www.holycross.edu/), USA
-* Moshe Berman, [Brooklyn College](http://www.brooklyn.cuny.edu/), USA
+* Kevin Walsh, [College of the Holy Cross](https://www.holycross.edu/), USA
+* Moshe Berman, [Brooklyn College](https://www.brooklyn.cuny.edu/), USA
 * Theldo Cruz Franqueira,
   [Pontifícia Universidade Católica de Minas Gerais](https://www.pucminas.br/destaques/Paginas/default.aspx), Brazil
-* Theo Kluter, [Bern University of Applied Sciences](http://www.microlab.ch/), Switzerland
-* [Marcin Orlowski](http://MarcinOrlowski.com/), Poland
+* [Theo Kluter](https://www.bfh.ch/en/theo-kluter),
+  [Berner Fachhochschule | Haute école spécialisée bernoise](https://bfh.ch/electrical), Switzerland
+* [Torsten Maehne](https://www.bfh.ch/en/torsten-maehne),
+  [Berner Fachhochschule | Haute école spécialisée bernoise](https://bfh.ch/electrical), Switzerland
+* [Marcin Orlowski](http://www.marcinorlowski.com/), Poland
 * [Tom Niget](https://github.com/zdimension/), [LEAT](https://leat.univ-cotedazur.fr/),
   [Polytech Nice-Sophia](https://polytech.univ-cotedazur.fr/), France
 * [Liu Yuchen](https://github.com/smallg0at),
```

Comment:
Maybe I misunderstand the sort, but shouldn't Marcin come before Moshe, Tom before Torsten, and Liu before Marcin?

Reply:
I wasn’t sure about the sorting criteria. Should it be sorted by first name, last name, institution, or [number/amount/regularity of contributions](https://github.com/logisim-evolution/logisim-evolution/graphs/contributors)?

What do you think @BFH-ktt1?

Reply:
Sorting by last name would be also my preference to avoid too much discussion. I‘ll try to find some time this evening to update this PR.

Reply:
I also favor sorting by last name. But I know that at some point someone sorted them by the strings (so first name) which is how my name moved to the beginning.

I prefer them to have an obvious sort so we are not suggesting any other criteria. It did not occur to me that it was sorted by country, since USA was first.

Reply:
Now sorted by last name. Thank you, @maehne.